README.md

```diff
@@ -100,6 +48,47 @@ mutant --include lib --require virtus --use rspec Virtus::Attribute.build
 mutant --include lib --require virtus --use rspec Virtus::Attribute#type
 ```
 
+Configuration
+-------------
+
+Occasionally mutant will produce a mutation with an infinite runtime.  When this happens
```

Comment:
This is probably a hold-over habit from [typing on typewriters](http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period). I catch myself doing the same thing sometimes even though I never really typed on a typewriter much as a kid.